fixing minor grammar and typos (#751)
zkSNACKs/WasabiDoc: docs/why-wasabi/LegalConcern.md

---

> Using Bitcoin and other cryptocurrencies in the everyday course of businesses are becoming more and more popular.

Reviewer: I think `is` is correct here but I am not sure.
How about something like this:
```suggestion
Bitcoin and other cryptocurrencies usage in the everyday course of businesses is becoming more and more popular.
```

Reviewer 2: It is `is`, not `are`.

Author: Indeed, you're right, my bad!

As each and every price discount is negotiated on a case by case basis, considering factors as creditworthiness and timing of payment, transparent supplier prices would only cause enormous pressure on suppliers to deal with negotiations.

Beyond the price of products, the timing of payments is also fully transparent on the blockchain.

> However, in certain circumstances, it is also a crucial business secret to keep the information on the sequence of invoice payments indoors.

Reviewer 2: I think `of` is correct here?

Author: I think `of` works too, but I think it's more appropriate to use `on` or `about`
When using `about`, then `the` should be left out:
"To keep information about the sequence of invoice payments indoors."
